### ISSUE TITLE
whatsapp-web.js crash right after scan qr code

### ISSUE THREAD

**Issue by userbox020** [pedroslopez/whatsapp-web.js#3075]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

I run the example code and get the follow error

```
node:internal/process/promises:289
            triggerUncaughtException(err, true /* fromPromise */);
            ^

Error [TypeError]: Cannot read properties of undefined (reading 'default')
    at evaluate (evaluate at Client.initialize (C:\Users\admin\Desktop\www\whatsapp-web-v3\whatsapp-web.js\src\Client.js:340:20), <anonymous>:4:94)
    at #evaluate (C:\Users\admin\Desktop\www\whatsapp-web-v3\whatsapp-web.js\node_modules\puppeteer-core\lib\cjs\puppeteer\cdp\ExecutionContext.js:376:56)
    at async ExecutionContext.evaluate (C:\Users\admin\Desktop\www\whatsapp-web-v3\whatsapp-web.js\node_modules\puppeteer-core\lib\cjs\puppeteer\cdp\ExecutionContext.js:271:16)
    at async IsolatedWorld.evaluate (C:\Users\admin\Desktop\www\whatsapp-web-v3\whatsapp-web.js\node_modules\puppeteer-core\lib\cjs\puppeteer\cdp\IsolatedWorld.js:99:16)
    at async CdpFrame.evaluate (C:\Users\admin\Desktop\www\whatsapp-web-v3\whatsapp-web.js\node_modules\puppeteer-core\lib\cjs\puppeteer\api\Frame.js:342:20)
    at async CdpPage.evaluate (C:\Users\admin\Desktop\www\whatsapp-web-v3\whatsapp-web.js\node_modules\puppeteer-core\lib\cjs\puppeteer\api\Page.js:823:20)
    at async Client.initialize (C:\Users\admin\Desktop\www\whatsapp-web-v3\whatsapp-web.js\src\Client.js:340:9)

Node.js v21.6.2
```

### Expected behavior

run as normal

### Steps to Reproduce the Bug or Issue

```
git clone https://github.com/pedroslopez/whatsapp-web.js.git

cd whatsapp-web.js

npm install redis
npm install ioredis
npm install qrcode-terminal

npm install whatsapp-web.js
npm install github:pedroslopez/whatsapp-web.js#webpack-exodus

npm install puppeteer@latest

node example.js
```

### Browser Type

Google Chrome

### WhatsApp Account Type

Standard

### Does your WhatsApp account have multidevice enabled?

Yes, I am using Multi Device

### Environment

Windows Server 2022 Standard
Node.js v21.6.2

**Comment by zieru** [comment]
me too, my whatsapp client stop working too,

Environment
Ubuntu 18
Node.js v20

**Comment by Hambaini** [comment]
i have same problem.please help guys ..

**Comment by Thiago1999** [comment]
Encontro o mesmo problema aqui...

**Comment by indrafani03** [comment]
i have same problem.please help guys ..

**Comment by felipeDS91** [comment]
I had the same problem here and I fix it using 1.23.1-alpha.6 + WebCache: 2.24xx

**Comment by alechkos** [comment]
#2789 
#2816
